test/src/test/java/hudson/model/QueueTest.java

```diff
@@ -100,6 +105,16 @@
 
     @Rule public JenkinsRule r = new NodeProvisionerRule(-1, 0, 10);
 
+    private File queueFile;
+
+    @Before
+    public void resetQueueState() throws IOException {
+        queueFile = new File(r.jenkins.getRootDir(), "queue.xml");
+        XmlFile xmlFile = new XmlFile(Queue.XSTREAM, queueFile);
+        xmlFile.write(new Queue.State());
+        r.jenkins.getQueue().load();
```

Comment:
Then it ought to be made into an instance field of `Queue`. We do not want `static` state.